doc/modules/tree.rst

```diff
@@ -375,6 +375,10 @@ Tips on practical use
     predict. Training time can be orders of magnitude faster for a sparse
     matrix input compared to a dense matrix when features have zero values in
     most of the samples.
+    
+  * In case that there are multiple classes with the same and highest
+    probability, the classifier will predict the class with the lowest index
+    amongst those classes.
```

Comment:
```suggestion
  * In case that there are multiple classes with the same and highest
    probability, the classifier will predict the class with the lowest index
    amongst those classes in `clf.classes_`.
```

I'm not sure this fits in a "tips on practical use" section. I think we should rather put this above, just above the sentence "Alternatively, the probability of each class can be predicted..."

(I would also remove "Alternatively" which may cause confusion with the new sentence added in-between).

Reply:
Maybe instead of `clf.classes_`, we can use `:term:`\``classes_`\` to associate the term with the glossary.

WDYT @NicolasHug?

Reply:
I would agree with this @alfaro96